docs/language/accounts.mdx

```diff
@@ -359,6 +359,14 @@ to all its stored objects.
 
   The path must be a storage path, i.e., only the domain `storage` is allowed.
 
+- `cadence•fun typeAt(_ path: StoragePath): Type?`
```

Comment:
I named it `typeAt` to avoid confusion with the `Type` function, which is used to create a runtime type. They are slightly different, as one has a capital letter and the other doesn't, but I still think it might be confusing to have both of these builtins have the same name.

Reply:
Hmm, I hadn't thought of that at all. Given that this is a function on `AuthAccount`, I don't really think it'll be confused with functionality for run-time types